backend/api/Services/InspectionFindingsService.cs

```diff
@@ -0,0 +1,65 @@
+﻿using Api.Database.Context;
+using Microsoft.EntityFrameworkCore;
+
+
+namespace Api.Services
+{
+    public class InspectionFindingService(
+        ILogger<InspectionFindingService> logger,
+        IServiceScopeFactory serviceScopeFactory,
+        IHostApplicationLifetime hostApplicationLifetime,
+        IConfiguration configuration) : IHostedService
+    {
+        private readonly ILogger<InspectionFindingService> _logger = logger;
+        private readonly IServiceScopeFactory _serviceScopeFactory = serviceScopeFactory;
+        private readonly IHostApplicationLifetime _hostApplicationLifetime = hostApplicationLifetime;
+        private readonly TimeSpan _timeSpan = configuration.GetValue<TimeSpan>("InspectionFindingEventHandler:TimeSpan");
+
+        public Task StartAsync(CancellationToken cancellationToken)
+        {
+            return Task.CompletedTask;
+        }
+
+        public Task StopAsync(CancellationToken cancellationToken)
+        {
+            return Task.CompletedTask;
+        }
```

Comment:
StartAsync and StopAsync methods simply return a completed task, indicating that the service has started or stopped successfully. They called by EventHandler in Subscribe and Unsubscribe asnd those methods are required in EventHandler because they are the abstract methods of the inherited class.

Reply:
Done.